third_party/amd/include/TritonAMDGPUToLLVM/Passes.h

```diff
@@ -38,7 +38,8 @@ createConvertBuiltinFuncToLLVMPass(bool ftz);
 std::unique_ptr<OperationPass<ModuleOp>>
 createTritonAMDGPUInsertInstructionSchedHintsPass();
 std::unique_ptr<OperationPass<ModuleOp>>
-createTritonAMDGPULowerInstructionSchedHintsPass(int32_t numStages,
+createTritonAMDGPULowerInstructionSchedHintsPass(std::string arch,
```

Comment:
Nit: `StringRef` instead of `std::string` to avoid copy.

Reply:
Done